[2025-03-30T18:15:33.174+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-03-30T18:15:33.189+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_deliverable.create_hive_table manual__2025-03-30T18:15:19.927483+00:00 [queued]>
[2025-03-30T18:15:33.192+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_deliverable.create_hive_table manual__2025-03-30T18:15:19.927483+00:00 [queued]>
[2025-03-30T18:15:33.193+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-03-30T18:15:33.206+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): create_hive_table> on 2025-03-30 18:15:19.927483+00:00
[2025-03-30T18:15:33.212+0000] {standard_task_runner.py:64} INFO - Started process 185 to run task
[2025-03-30T18:15:33.215+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'dag_deliverable', 'create_hive_table', 'manual__2025-03-30T18:15:19.927483+00:00', '--job-id', '177', '--raw', '--subdir', 'DAGS_FOLDER/dag_deliverable.py', '--cfg-path', '/tmp/tmp6e2bo7c0']
[2025-03-30T18:15:33.216+0000] {standard_task_runner.py:91} INFO - Job 177: Subtask create_hive_table
[2025-03-30T18:15:33.240+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_deliverable.create_hive_table manual__2025-03-30T18:15:19.927483+00:00 [running]> on host airflow
[2025-03-30T18:15:33.293+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='dag_deliverable' AIRFLOW_CTX_TASK_ID='create_hive_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-30T18:15:19.927483+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-30T18:15:19.927483+00:00'
[2025-03-30T18:15:33.293+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-03-30T18:15:33.307+0000] {base.py:84} INFO - Using connection ID 'hive_default' for task execution.
[2025-03-30T18:15:33.328+0000] {hive.py:475} INFO - USE `default`
[2025-03-30T18:15:33.347+0000] {hive.py:475} INFO - DROP TABLE IF EXISTS weather.sensor_data
[2025-03-30T18:15:33.596+0000] {hive.py:475} INFO - 
            CREATE EXTERNAL TABLE IF NOT EXISTS weather.sensor_data (
                record_timestamp DATE,
                temperature_salon FLOAT,
                humidity_salon FLOAT,
                air_salon FLOAT,
                temperature_chambre FLOAT,
                humidity_chambre FLOAT,
                air_chambre FLOAT,
                temperature_bureau FLOAT,
                humidity_bureau FLOAT,
                air_bureau FLOAT,
                temperature_exterieur FLOAT,
                humidity_exterieur FLOAT,
                air_exterieur FLOAT
            )
            ROW FORMAT SERDE 'org.apache.hive.hcatalog.data.JsonSerDe'
            STORED AS TEXTFILE
            LOCATION 'hdfs://namenode:9000/topics/sensores/partition=0'
        
[2025-03-30T18:15:33.677+0000] {dag_deliverable.py:134} INFO - âœ… Tabla creada correctamente en Hive
[2025-03-30T18:15:33.678+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-03-30T18:15:33.679+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-03-30T18:15:33.684+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_deliverable, task_id=create_hive_table, run_id=manual__2025-03-30T18:15:19.927483+00:00, execution_date=20250330T181519, start_date=20250330T181533, end_date=20250330T181533
[2025-03-30T18:15:33.748+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-03-30T18:15:33.759+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-30T18:15:33.765+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
